pkg/transforms/http.go

```diff
@@ -209,7 +212,33 @@ func (sender HTTPSender) httpSend(ctx interfaces.AppFunctionContext, data interf
 		return true, data
 	}
 
-	ctx.LoggingClient().Debugf("Sent %d bytes of data in pipeline '%s'. Response status is %s", len(exportData), ctx.PipelineId(), response.Status)
+	// capture the size into metrics
+	exportDataBytes := len(exportData)
+	// TODO: EdgeX 3.0 refactor size metrics once receivers are pointers (like mqtt size metrics)
+	metrics := gometrics.DefaultRegistry.Get(internal.HttpExportSizeName)
+	var httpMetrics gometrics.Histogram
+	if metrics == nil {
+		var err error
+		lc.Debugf("Initializing metric %s.", internal.HttpExportSizeName)
+		httpMetrics = gometrics.NewHistogram(gometrics.NewUniformSample(internal.MetricsReservoirSize))
```

Comment:
A bit more explicit... 

```suggestion
		httpExportSizeMetric = gometrics.NewHistogram(gometrics.NewUniformSample(internal.MetricsReservoirSize))
```